.gitignore

```diff
@@ -10,3 +10,4 @@ npm-debug.log
 # other
 .next
 .DS_Store
+.idea
```

Comment:
Maybe we should have a seperate PR which puts that advice at the top of the .gitignore?